src/redux/constants.js

```diff
@@ -8,6 +8,7 @@ export const runtimeStages = {
   PARSING: 'parsing',
   COMPILING: 'compiling',
   EXECUTING: 'executing',
+  PAUSING: 'pausing',
```

Comment:
Since we had the action type, I went ahead and added it in. I'm not sure if that's actually something we should do though, perhaps we should remove it altogether.